age/front/src/components/contractInteraction.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ChangeEvent, useState } from "react";
+import { changeAge, getAge } from "../utils/age";
+import { IAccount } from "@massalabs/wallet-provider";
+
+interface ContractInteractionProps {
+    account: IAccount;
+}
+
+export default function ContractInteraction({
+    account,
+}: ContractInteractionProps) {
+    const [name, setname] = useState<string>("");
```

Comment:
I would have set a different variable for set and get.
Here if we update the input in get age it also update the set name input.

Reply:
yes, done